doc/source/cookbook/extract_fixed_resolution_data.py

```diff
@@ -30,4 +30,4 @@
 
 # If we want to then access this datacube in the h5 file, we can now...
 f = h5py.File("my_data.h5", "r")
-print(f["density"].value)
+print(f["density"][()])
```

Comment:
I know this is how it gets done nowadays, but I think it might be more in keeping with the style if we do a colon.

Reply:
This is the recommended syntax from `h5py` that was included in the deprecation warning for `.value`.

Reply:
I should also say that these changes are just cherry-picked from a PR already merged into master.

Reply:
OK, comment retracted!